.github/workflows/jitpack_snapshot_commit.yml

```diff
@@ -0,0 +1,50 @@
+name: JitPack Snapshot & Commit Build
+
+on:
+  pull_request:
+    branches:
+      - master
+      -
```

Comment:
Is this line correct ? Looks wrong to me.

Reply:
Shouldn't Matter, but fixed